elements/lisk-tree/src/sparse_merkle_tree/types.ts

```diff
@@ -12,6 +12,22 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+export interface InclusionProof {
+	readonly siblingHashes: Buffer[];
+	readonly queries: InclusionProofQuery[];
+}
+
+export interface InclusionProofQuery {
+	readonly key: Buffer;
+	readonly value: Buffer;
+	readonly bitmap: Buffer;
+}
+
+export interface InclusionProofQueryWithHash extends InclusionProofQuery {
+	hash: Buffer;
+	binaryBitmap: string;
+}
```

Comment:
Do we really need this ?

I think `hash` & `binaryBitmap` can be derived from the base properties using other util functions.

Reply:
I am not sure, as per the LIP these attributes will be passed to the function.